mdpow/workflows/dihedrals.py

```diff
@@ -157,110 +180,151 @@ def rdkit_conversion(u, resname):
 
     """
 
-    try:
-        solute = u.select_atoms(f'resname {resname}')
-        mol = solute.convert_to('RDKIT')
-    except AttributeError:
-        elements = [guess_atom_element(name) for name in u.atoms.names]
-        u.add_TopologyAttr("elements", elements)
-        solute = u.select_atoms(f'resname {resname}')
-        mol = solute.convert_to('RDKIT')
+    elements = [guess_atom_element(name) for name in u.atoms.names]
+    u.add_TopologyAttr("elements", elements)
+
+    solute = u.select_atoms(f'resname {resname}')
+    solute.unwrap(compound='residues', reference='com')
+
+    mol = solute.convert_to('RDKIT')
```

Comment:
Did you resort to always guessing because the elements are never present?

If so, write a comment stating this assumption and _change the docstring_ — it does not agree anymore with the code.

Otherwise the try/except is cleaner because it uses user data and only resorts to guessing when necessary.

Reply:
I cannot find the reason why it was removed, so I will revert to try/except and see how things go.

Reply:
It may have had something to do with our data that led me to believe that RDKit conversion always required this, rendering try/except useless. We should look into it more, but having the try/except present in the meantime hurts none at all.